Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -97,7 +98,12 @@ protected virtual void InitPointer()
 
         protected virtual void SetPlayAreaCursorTransform(Vector3 destination)
         {
-            playAreaCursor.transform.position = destination;
+            playArea = GameObject.FindObjectOfType<SteamVR_PlayArea>();
```

Comment:
I would recommend doing this assignment in the Start() or Awake() method to reduce the need for it to be called every time the cursor is drawn (reduces overhead on a costly Find call).

If it doesn't work in Start() or Awake() then maybe do a check to see if it's already been set:

```
if (playArea == null) {
  playArea = GameObject.FindObjectOfType<SteamVR_PlayArea>();
}
```